modules/cli/pages/cbstats/cbstats-scopes-details.adoc

```diff
@@ -22,72 +22,90 @@ The optional `vbucket-reference` parameter is an integer, in the range of `0` to
 For common [.cmd]`cbstats` options, see xref:cli:cbstats-intro.adoc[cbstats].
 
 == Examples
 
-The following command retrieves scopes-details on all vBuckets for `testBucket`:
+The following command retrieves scopes-details on all vBuckets for `travel-sample`, for the specified node:
 
 ----
-/opt/couchbase/bin/cbstats -u Administrator -p password \
--b testBucket localhost:11210 scopes-details
+/opt/couchbase/bin/cbstats localhost:11210 \
+-u Administrator -p password \
+-b travel-sample scopes-details
 ----
 
 The initial section of the output is as follows:
 
 ----
-manifest:scopes:                         2
-manifest:scopes:0x0:collections:         2
-manifest:scopes:0x0:name:                _default
-manifest:scopes:0x8:collections:         2
-manifest:scopes:0x8:name:                my_scope
-manifest:uid:                            4
-vb_0:manifest:scope:0x0:collection:0:    0xa
-vb_0:manifest:scope:0x0:collection:2:    0x0
-vb_0:manifest:scope:0x8:collection:1:    0x9
-vb_0:manifest:scope:0x8:collection:3:    0x8
-vb_0:manifest:scopes:                    2
-vb_0:manifest:scopes:0:                  0x8
-vb_0:manifest:scopes:1:                  0x0
-vb_1:manifest:scope:0x0:collection:0:    0xa
-vb_1:manifest:scope:0x0:collection:2:    0x0
-vb_1:manifest:scope:0x8:collection:1:    0x9
-vb_1:manifest:scope:0x8:collection:3:    0x8
+0x0:0x0:name:          _default
+0x0:collections:       1
+0x0:name:              _default
+0x8:0x9:name:          MyCollection
+0x8:collections:       1
+0x8:name:              MyScope
+0x9:0xa:name:          MySecondCollection
+0x9:0xb:name:          MyThirdCollection
+0x9:collections:       2
+0x9:name:              MySecondScope
+manifest_uid:          7
+vb_0:0x0:              2
+vb_0:0x0:0x0:items:    38
+vb_0:0x8:              1
+vb_0:0x8:0x9:items:    0
+vb_0:0x9:              0
+vb_0:0x9:0xa:items:    0
+vb_0:0x9:0xb:items:    0
+vb_0:manifest_uid:     7
+vb_0:scopes:           3
+vb_1:0x0:              2
+vb_1:0x0:0x0:items:    29
       .
       .
       .
 ----
 
-The initial six lines provide information that identifies existing scopes.
-A total of `2` are listed, including the `_default` scope.
-Each is identified with a hexadecimal number: for example, the `_default` scope is identified as `0x0`.
+Each line of the output presents a data _key_.
+Each key is either of the format `scopeID` &#58; `collectionID` &#58; `datumLabel`; or of the format `scopeID` &#58; `datumLabel`.
+Each `scopeID` or `collectionID` is a hexadecimal number, prefixed by `0x`.
+Each `datumLabel` is a string.
+Each data key is concluded with a colon, and the associated value is presented to the right of the colon.
 
-Subsequent information in the output relates to each vBucket corresponding to `testBucket`.
+The output's initial lines describe the scopes and collections in the bucket on the specified node.
```

Comment:
As per other examples, "In this example.."

Reply:
ok.